.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,12 +47,22 @@ Enter your extended release note in the block below. If the PR requires addition
 
 <!--
 This section can be blank if this pull request does not require a release note.
-Please use the following format for linking documentation or pass the
-section below:
+
+The Kubernetes Release Notes website (https://relnotes.k8s.io) is able to
+provide additional documentation to the end users, which should help them to
+retrieve further documentation resources about the change in this PR.
```

Comment:
This seems kind of redundant. Maybe something like "... able to provide additional documentation to the end users about the changes in this PR."

Reply:
Okay, changed as suggested.